src/helpers/url.ts

```diff
@@ -16,5 +17,22 @@ export const normalizeAllUrls = (data: Event): Event => {
 export const getFlexibleLineFromPath = (
   flexibleLines: FlexibleLine[],
   params: Params,
-): FlexibleLine | undefined =>
-  flexibleLines.find((flexibleLine) => flexibleLine.id === params.id);
+): FlexibleLine | undefined => {
+  const line = flexibleLines.find(
+    (flexibleLine) => flexibleLine.id === params.id,
+  );
+
+  // Adding a "key" to each stop point
+  return line
+    ? {
+        ...line,
+        journeyPatterns: line.journeyPatterns?.map((jp) => ({
+          ...jp,
+          pointsInSequence: jp.pointsInSequence.map((pis) => ({
+            ...pis,
+            key: createUuid(),
```

Comment:
Adding this key was essential to avoid weird things like in https://github.com/entur/enki/issues/1487

Reply:
A more appropriate place to do this is in the fleixlbeLines reducer: https://github.com/entur/enki/blob/master/src/reducers/flexibleLines.ts#L12

Reply:
Thanks, will move it 👍